conftest.py

```diff
@@ -707,6 +707,14 @@ def get_inspect_command_namespace_string(node: Node, test_name: str) -> str:
     return namespace_str
 
 
+def calculate_must_gather_timer(test_start_time):
+    if test_start_time > 0:
+        return int(datetime.datetime.now().strftime("%s")) - test_start_time
+    else:
+        LOGGER.warning(f"Could not get start time of test. Collecting must-gather for {TIMEOUT_5MIN}s")
```

Comment:
```suggestion
        LOGGER.warning(f"Could not get start time of test. Collecting must-gather for last {TIMEOUT_5MIN}s")
```

Reply:
it was just a copy-past
anyway, thanks - updated it